react/admin/OrganizationsTable.tsx

```diff
@@ -45,6 +47,14 @@ const OrganizationsTable = () => {
             <PageHeaderTitle>
               {formatMessage(messages.tablePageTitle)}
             </PageHeaderTitle>
+
+            <PageHeaderActions>
+              <Route
+                path="/organizations"
+                exact
+                component={CreateOrganizationButton}
```

Comment:
This makes sure to only show the button when the user is at the organization page, just like before.